bin/tessel-2.js

```diff
@@ -70,6 +71,30 @@ parser.command('install-drivers')
   })
   .help('Install drivers');
 
+parser.command('crash-reporting-off')
+  .callback(function() {
+    crashReporter.turnOff()
+      .then(module.exports.closeSuccessfulCommand, module.exports.closeFailedCommand);
+  })
+  .help('Turn off crash reporting completely');
+
+parser.command('crash-reporting-on')
```

Comment:
This should be `t2 crash-reporter` with on and off flags, like all the others. Here's wifi:

```Js   
  .option('off', {
    flag: true,
    help: 'Disable the wireless network'
  })
  .option('on', {
    flag: true,
    help: 'Enable the wireless network'
  })
```